src/main/java/StringAddCal.java

```diff
@@ -0,0 +1,52 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCal {
+    public static int splitAndSum(String input) throws RuntimeException {
+        if (nullOrEmpty(input) == false) {
+         return 0;
+        }
+
+        String[] split = customDelimeter(input);
+
+        return calAllSum(split);
+    }
+
+    public static String[] customDelimeter(String input) {
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(input);
+        if(m.find()) {
+            String customDelimeter = m.group(1);
+            String[] tokens = m.group(2).split(customDelimeter);
+            return tokens;
+        }
+        return input.split(",|:");
+    }
+
+    public static int stringCastingInt(String input) {
+        int i = Integer.parseInt(input);
+        return i;
+    }
+
+    public static void negative(int number) {
+        if(number < 0) {
+            throw new RuntimeException();
+        }
+    }
+
+    public static boolean nullOrEmpty(String input) {
+        if(input == null || input.isEmpty()) {
+            return false;
+        }
+        return true;
```

Comment:
메서드 이름을 보면 수식이 빈 문자열 또는 null인지를 확인한다는 의미이니, 실제로 수식이 빈 문자열이거나 null이라면 반환값이 true가 되어야 하지 않을까요?
```suggestion
        return input == null || input.isEmpty();
```

Reply:
수정 완료